soc/arm/st_stm32/common/soc_config.c

```diff
@@ -26,11 +26,13 @@
 static int st_stm32_common_config(const struct device *dev)
 {
 #ifdef CONFIG_LOG_BACKEND_SWO
-	/* TRACE pin assignment for asynchronous mode */
-	DBGMCU->CR &= ~DBGMCU_CR_TRACE_MODE_Msk;
-	/* Enable the SWO pin */
-	DBGMCU->CR |= DBGMCU_CR_TRACE_IOEN;
+	/* Enable SWO trace asynchronous mode */
+#if defined(CONFIG_SOC_SERIES_STM32WBX)
+	LL_DBGMCU_EnableTraceClock();
```

Comment:
Open question:
I don't know much on this part, but this functions is also defined on U5 and L5, as well as `LL_DBGMCU_SetTracePinAssignment`.
So for these 2 series, is `LL_DBGMCU_SetTracePinAssignment()` a better choice than `LL_DBGMCU_EnableTraceClock(LL_DBGMCU_TRACE_ASYNCH)`?